Utilities/bootstrap

```diff
@@ -129,8 +131,10 @@ if platform.system() == 'Darwin':
 
 if platform.system() == 'Linux':
     g_shared_lib_ext = ".so"
-else:
+elif platform.system() == 'Darwin':
     g_shared_lib_ext = ".dylib"
+elif platform.system() == 'Windows':
+    g_shared_lib_ext = ".dll"
```

Comment:
```
else:
   raise RuntimeError(...)
```
?

Reply:
Or, do something more sensible and just move "Linux" to the "else" case... there's plenty of other OSes out there and virtually all of them use .so as the shared library suffix. It's Darwin and Windows which are the exceptional cases.